docs/admin/region-availability.md

```diff
@@ -34,7 +37,65 @@ For more information about purchasing a Fabric subscription, see [Buy a Microsof
 | West US 3         | UK South            |             |                    |                    |
 |                   | UK West             |             |                    |                    |
 
+# Power BI F SKU Regional Availability
+
+New regions or constrained regions will   only support Power BI artifacts. Over time, the new regions will expand to include additional Fabric artifacts. Currently, the regions listed below support Power BI and are in the process of onboarding additional Fabric artifacts. Note that India West is capacity constrained and will exclusively host Power BI on the F SKU and not launch other Fabric features.
+
+| Americas      | Europe       | Middle East               | Africa           | Asia Pacific             |
+|:--------------|:-------------|:--------------------------|:-----------------|:-------------------------|
+| Mexico Central| France South | Israel Central            | South Africa West| India West (Indefinitely)|
+|               | Germany North| Qatar Central (until 2027)|                  | Japan West               |
+|               | Norway West  | UAE Central               |                  | Korea South              |
+|               | Spain Central|                           |                  |                          |
+
+# Public Preview Feature Regional Availability
```

Comment:
`## Preview feature regional availability`

Reply:
This change is fine.